AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomGPU_AreaLightScreenshotTest.py

```diff
@@ -180,13 +201,20 @@ def AtomGPU_LightComponent_AreaLightScreenshotsMatchGoldenImages():
                 AtomComponentProperties.light('Light type')) == LIGHT_TYPES['sphere'])
 
         # 14. Enter game mode and take a screenshot then exit game mode.
-        enter_exit_game_mode_take_screenshot("AreaLight_5.ppm", Tests.enter_game_mode, Tests.exit_game_mode)
+        enter_exit_game_mode_take_screenshot(screenshot_names[4], Tests.enter_game_mode, Tests.exit_game_mode)
 
         # 15. Delete the Area Light entity.
         area_light_entity.delete()
         Report.result(Tests.area_light_entity_deleted, not area_light_entity.exists())
 
-        # 16. Look for errors.
+        # 16. Compare screenshots to golden images.
+        for screenshot_name, screenshot_threshold in zip(screenshot_names, screenshot_thresholds):
+            compResult = compare_screenshot_to_golden_image(FOLDER_PATH, screenshot_name, screenshot_name)
+            Report.result(("Screenshot comparison succeeded.", "Screenshot comparison failed."), compResult.result_code == azlmbr.utils.ImageDiffResultCode_Success)
+            if compResult.result_code == azlmbr.utils.ImageDiffResultCode_Success:
+                Report.result((f"{screenshot_name} diff score {compResult.diff_score} under threshold {screenshot_threshold}.", f"{screenshot_name} diff score {compResult.diff_score} over threshold {screenshot_threshold}."), compResult.diff_score < screenshot_threshold)
```

Comment:
line length here is making it hard to see the threshold in action
declare your tuple outside of the Report.result call:
```suggestion
                screen_compare_test = (
                    f"{screenshot_name} diff score {compResult.diff_score} under threshold {screenshot_threshold}.",
                    f"{screenshot_name} diff score {compResult.diff_score} over threshold {screenshot_threshold}.")
                Report.result(screen_compare_test, compResult.diff_score < screenshot_threshold)
```